dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -215,7 +215,7 @@
       </AdditionalIncludeDirectories>
       <AdditionalOptions>/std:c++17 /bigobj %(AdditionalOptions)</AdditionalOptions>
       <AdditionalOptions>/Wv:18 /Zm1000 %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalOptions>%(AdditionalOptions) /await</AdditionalOptions>
+      <AdditionalOptions Condition="'$(Configuration)'=='Release'">%(AdditionalOptions) /d2FH4</AdditionalOptions>
```

Comment:
>Configuration [](start = 39, length = 13)

Why only do this for Release?

Reply:
Oh, missed your note in the PR description.

---
In reply to: [304168725](https://github.com/microsoft/microsoft-ui-xaml/pull/1064#discussion_r304168725) [](ancestors = 304168725)